### PR TITLE
Update installer options for v 2.2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Review by tbrisker — APPROVED** [review]
thanks @upadhyeammit

**Review by tbrisker — CHANGES_REQUESTED** [review]
actually, we probably want to copy these changes to the nightly manual as well

File: _includes/manuals/2.2/3.2.2_installer_options.md [hunk 1]
```diff
@@ -177,6 +181,10 @@ More information about compute resources can be found in the [Compute Resources
         <td style='white-space:nowrap'>--[no-]enable-foreman-plugin-snapshot-management</td>
         <td>Enable 'foreman_plugin_snapshot_management' puppet module (<a href="https://github.com/ATIX-AG/foreman_snapshot_management">foreman_snapshot_management</a>)</td>
       </tr>
+      <tr>
+        <td style='white-space:nowrap'>--[no-]enable-foreman-plugin-statistics</td>
+        <td>Enable 'foreman_plugin_statistics' puppet module (<a href="/plugins/foreman_statistics">foreman_statistics</a>)</td>
```
Comment (ekohl) [inline _includes/manuals/2.2/3.2.2_installer_options.md:186]: I'm now remembering I wrote the code for this and happy that it still works. Note how it's the Foreman manual vs the GH project :)

**Review by ekohl — COMMENTED** [review]
Looks good, but as @tbrisker said: nightly deserves the same.

**Comment by upadhyeammit** [comment]
> actually, we probably want to copy these changes to the nightly manual as well
I forgot most of the time, will keep this in mind when updating latest stable next time.

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Pull request merged**